src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.length > 1 && style.includes(':'));
```

Comment:
here u don't need an extra iteration (filter method).
u can check ur `value` on each iteration in ur `reduce` method.
I'll leave comments below
```suggestion
```